### PR TITLE
Add new rules

### PR DESCRIPTION
**Pull request by loverajoel**
## Update project guidelines

Based on comments in this [issue](https://github.com/loverajoel/jstips/issues/251) I rewrite the rules.

Please feel free to add suggestions!

### CODE REVIEW

**Comment by kurtextrem** [comment]
:+1: 
By the way, now we can add issue and PR templates. Is that something we want?

**Comment by loverajoel** [comment]
@kurtextrem I added yesterday the PR template

**Comment by kurtextrem** [comment]
Aah sorry :)
Am 19.02.2016 17:58 schrieb "Joel Lovera" notifications@github.com:

**Comment by loverajoel** [comment]
@zenopopovici haha 5 is better to me, we can try with 5 and if we see that it is much we can return to 3

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ![header](https://raw.githubusercontent.com/loverajoel/jstips/gh-pages/resources/jstips-header-blog.gif)
 
-# Introducing JavaScript Tips [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
-> New year, new project. **A JS tip per day!**
+# JS Tips [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+> A useful Js tip
```
Comment (drakmail) [inline README.md]: Maybe `A useful JS tips`?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ![header](https://raw.githubusercontent.com/loverajoel/jstips/gh-pages/resources/jstips-header-blog.gif)
 
-# Introducing JavaScript Tips [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
-> New year, new project. **A JS tip per day!**
+# JS Tips [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+> A useful Js tip 
 
-With great excitement, I introduce these short and useful daily JavaScript tips that will allow you to improve your code writing. With less than 2 minutes each day, you will be able to read about performance, conventions, hacks, interview questions and all the items that the future of this awesome language holds for us.
+This is an awesome project about shorts and useful JavaScript tips that will allow you to improve your code writing. With less than 2 minutes, you will be able to read about performance, conventions, hacks, interview questions and all the items that the future of this awesome language holds for us.
```
Comment (drakmail) [inline README.md]: shorts -> short?

File: README.md [hunk 3]
```diff
@@ -81,4 +81,4 @@ There are a lot of ways to get update, choose your own
 - 00 - [Insert item inside an Array](https://github.com/loverajoel/jstips/blob/gh-pages/_posts/en/2015-12-29-insert-item-inside-an-array.md)
 
 ### License
-[![CC0](http://i.creativecommons.org/p/zero/1.0/88x31.png)](http://creativecommons.org/publicdomain/zero/1.0/)
+[GNU GENERAL PUBLIC LICENSE](https://github.com/loverajoel/jstips/blob/gh-pages/LICENCE.md)
```
Comment (drakmail) [inline README.md]: LICEN**S**E.md?

**Comment by zenopopovici** [comment]
@loverajoel There are still some spelling mistakes suggested by @drakmail. Take a look when you have the time.

**Comment by loverajoel** [comment]
@drakmail fiexed, thanks!

**Comment by drakmail** [comment]
:shipit:

**Comment by zenopopovici** [comment]
:shipit:

**Comment by bmkmanoj** [comment]
5 is good, :shipit:

**Comment by loverajoel** [comment]
Thanks guys!

**Comment by kurtextrem** [comment]
:tada: this is awesome :)

**Pull request merged**